server/db/models/conversation.js

```diff
@@ -17,9 +24,25 @@ Conversation.findConversation = async function (user1Id, user2Id) {
       }
     }
   });
-
   // return conversation or null if it doesn't exist
   return conversation;
 };
 
+
+Conversation.updateConversation = async function (id, user1UnreadMsg = 0, user2UnreadMsg = 0) {
+  const conversation = await Conversation.findByPk(id);
+  if (conversation) {
+    const values = {
+      user1Id: conversation.user1Id,
+      user2Id: conversation.user2Id,
+      user1UnreadMsg: user1UnreadMsg,
+      user2UnreadMsg: user2UnreadMsg
+    }
+    conversation.update(values).then(updatedRecord => {
+      return updatedRecord;
+    });
```

Comment:
Instead of mixing the `async/await` and `then/catch` syntax, I would suggest sticking with the `async/await` one since it matches the existing codebase.